code-challenges.js

```diff
@@ -31,9 +54,43 @@ var hand3 = [5, 5, 5, 5, 4]
 var hand4 = [3, 5, 3, 5, 3]
 // Expected output: true
 
+//Create a function that takes in an array
+const pokerFace = (array) => {
```

Comment:
haha definitely showing your pokerFace here but I don't think I see a full house. Feel free to let me know if you'd like to pair on this solution. Some comments for your general strategy would be great too if you are ready for coding it yet.

Reply:
There are iterative approaches to this solution but I'd encourage you to consider using a data structure that help model a full house.